lib/src/byte_collector.dart

```diff
@@ -46,7 +46,7 @@ T _collectBytes<T>(Stream<List<int>> source,
   var completer = Completer<Uint8List>.sync();
   var subscription =
       source.listen(bytes.add, onError: completer.completeError, onDone: () {
-    completer.complete(bytes.takeBytes());
+    completer.complete(bytes.toBytes());
```

Comment:
Try using a `BytesBuilder(copy: false)` instead, then use `takeBytes`.

Reply:
Using `copy: false` seems to fix it. I'll use that. Thanks!